docs/users-guide/08-dashboard-filters.md

```diff
@@ -71,24 +69,19 @@ In both cases, you can pick one or multiple selections for your filter.
 
 ![Multi-select](images/dashboard-filters/multi-select.png)
 
-If Metabase somehow picked the wrong behavior for your field, admins can go to the Data Model section of the admin panel and click on the gear icon by the field in question to manually choose between a list, a search box, or just a plain input box.
+If Metabase somehow picked the wrong behavior for your field, admins can go to the [Data Model](../administration-guide/03-metadata-editing.md) section of the admin panel and click on the gear icon by the field in question to manually choose between a list, a search box, or just a plain input box.
 
 ![Search options](images/dashboard-filters/search-options.png)
 
 ### Best practices
 
 Here are a few tips to get the most out of dashboard filters:
+
 - Try to keep the number of filters you add to a dashboard to two or three. This will make it easier for your teammates to quickly and easily understand what options are available to them when viewing your dashboard.
 - While you can add dashboard filters to a dashboard that already has a bunch of cards in it, it can be easier to start a new dashboard and think about what filters you intend to add to it, and then make sure that you only put cards in that dashboard that can be used with the filters.
 
-### Some things to keep in mind
```

Comment:
All of these are no longer relevant?

Reply:
No, not from what I can see.
- "When you activate..." - not using faded anymore
- "If you have a card..." - there's already a warning on combined cards, which I guess "multiple series" means?
It's actually the only one I was unsure of, but it needs to rewritten if it should stay.
- "While connecting cards to a filter..." - I have never seen that warning and can create a filter linked to Product.Category and People.Source.
- "You can’t use a dashboard filter..." - You can. It will just not replace the existing question filters making it return no data

Reply:
Maybe for that last bullet we could say something to the effect of

"If a question on a dashboard is already filtered by a field under the hood, hooking it up to a dashboard filter on that same field will cause it to return no data. For example if a question is called `Orders in January` which counts all the orders and has a filter on the `Date Order Was Placed` field to only select the orders placed January, hooking it up to a dashboard filter on `Date Order Was Placed` with different values will cause it to show no results as dashboard filters do no override question filters."

I also don't know how explicit we need to be about calling that out.

Reply:
If we want to call it out, then it should also include native queries (but that feels more obvious), though I don't remember ever seeing forum questions about this (or any of the bullet points, where most were incorrect anyway).

I think Maz was planning on some dashboard changes in upcoming versions, so I thought I would wait with the full rewrite until it could get Mazified.

Reply:
Yeah he and I have some thoughts on cleaning things up further. I think we can probably go with the removal for now and re-adjust based on any requests for clarification.